doc/roadmap.md

```diff
@@ -0,0 +1,19 @@
+# EaseAgent Roadmap
+
+## Features
+* Collecting the distributed trace data and metric data from a service perspective.
+* Support various third-party critical components in microservices.
```

Comment:
Support mainstream middlewares, libraries, and components in Java-based microservices

Reply:
fixed